packages/react-native-web/src/modules/ImageLoader/index.js

```diff
@@ -75,11 +82,15 @@ const requests = {};
 
 const ImageLoader = {
   abort(requestId: number) {
-    let image = requests[`${requestId}`];
-    if (image) {
+    const request = requests[`${requestId}`];
+    if (request) {
+      const { image, cleanup } = request;
+      if (cleanup) cleanup();
+
       image.onerror = null;
       image.onload = null;
-      image = null;
+      // Setting image.src to empty string aborts any ongoing image loading
```

Comment:
> Why does setting this to an empty string abort the load? Is this a standard behavior on all browsers at this point?

AFAIK it's mostly a vendor courtesy so it's not a standard behavior: https://stackoverflow.com/a/5278475/4834103
But I think the most popular browsers work that way

Reply:
Cool. Yeah I had some trouble finding details on this but ran into a pretty old article that suggested it would [break the entire internet](https://humanwhocodes.com/blog/2009/11/30/empty-image-src-can-destroy-your-site/) 😂